web/index.html

```diff
@@ -24,11 +25,11 @@ <h1 class="dib pa3 ma0 lh-tight">
     <section class="bg-near-white">
         <form id="queryForm" class="mw8 center lh-copy dark-gray br2 pv4 ph2 ph4-ns">
             <label class="db mt3 f6 fw6" for="multiaddr">Multiaddr</label>
-            <input class="db w-100 pa2" type="text" id="multiaddr" name="multiaddr" value="/ip4/1.2.3.4/tcp/1234/p2p/QmFoo"/>
+            <input class="db w-100 pa2" type="text" id="multiaddr" name="multiaddr" placeholder="/ip4/1.2.3.4/tcp/1234/p2p/QmFoo" required />
             <label class="db mt3 f6 fw6" for="cid">CID</label>
-            <input class="db w-100 pa2" type="text" id="cid" name="CID" value="QmData">
+            <input class="db w-100 pa2" type="text" id="cid" name="CID" placeholder="QmData" required>
             <label class="db mt3 f6 fw6" for="backendURL">Backend URL</label>
-            <input class="db w-100 pa2" type="text" id="backendURL" value="https://ipfs-check-backend.ipfs.io">
+            <input class="db w-100 pa2" type="text" id="backendURL" value="https://ipfs-check-backend.ipfs.io" placeholder="https://ipfs-check-backend.ipfs.io" required>

```

Comment:
This value needs to be passed in and the user rarely needs to change it (it's just to make it easy to point at another backend if ours is down or something). Will doing this make the user need to re-enter the data?

Maybe we leave this one alone, or we could just hard choice it into the JS or make it optional/implied and fix the JS to use it as the default.

Reply:
No is ok, the value is retained here. The placeholder is in addition, in case they delete the default, it will show them what the default used to be as a ghostly serving suggestion